sql/src/main/java/io/crate/execution/engine/collect/collectors/OrderedLuceneBatchIteratorFactory.java

```diff
@@ -54,7 +54,7 @@
 
     public static BatchIterator<Row> newInstance(List<OrderedDocCollector> orderedDocCollectors,
                                                  Comparator<Row> rowComparator,
-                                                 RowAccounting rowAccounting,
+                                                 RowAccountingWithEstimators rowAccounting,
```

Comment:
I think it would make sense to use the `RowAccounting` interface where possible instead of requiring the concrete implementation.